src/core/constants.js

```diff
@@ -696,6 +696,12 @@ export const BOTTOM = 'bottom';
  * @final
  */
 export const BASELINE = 'alphabetic';
+/**
```

Comment:
@davepagurek Does this look okay? I remembered from the 2.0 RFC that the plan is to use `Symbol` when the underlying value of a constant isn't needed. I'm guessing we just haven't gotten around to changing all the constants that fit that description?

Reply:
I think so!